<launch>

  <node pkg="zeta2_bringup" exec="zeta_if_node" name="zeta_if" output="screen">
    <param name="port_name" value="/dev/ttyUSB0"/>
    <param name="baudrate" value="115200"/>
  </node>
  
</launch>